docs/topics_5/web-openshift-troubleshoot.adoc

```diff
@@ -83,15 +83,15 @@ When accessing the executor Pod's logs the following error is seen.
 
```

Comment:
You can remove the ellipsis. When quoting logs, it is understood that they go on and on.